app/src/main/java/ch/hikemate/app/ui/saved/SavedHikesScreen.kt

```diff
@@ -52,27 +55,43 @@ fun SavedHikesScreen(
     savedHikesViewModel: SavedHikesViewModel = viewModel(factory = SavedHikesViewModel.Factory),
     navigationActions: NavigationActions
 ) {
-  // The Screen will need to be incorporated into the SideBarNavigation composable
-  SideBarNavigation(
+  BottomBarNavigation(
       onTabSelect = { route -> navigationActions.navigateTo(route) },
       tabList = LIST_TOP_LEVEL_DESTINATIONS,
       selectedItem = Route.SAVED_HIKES,
   ) { paddingValues ->
     var currentSection by remember { mutableStateOf(SavedHikesScreen.Planned) }
     val savedHikes by savedHikesViewModel.savedHike.collectAsState()
 
+    val pagerState = rememberPagerState { SavedHikesScreen.values().size }
+
     LaunchedEffect(Unit) { savedHikesViewModel.loadSavedHikes() }
 
+    LaunchedEffect(currentSection) { pagerState.animateScrollToPage(currentSection.ordinal) }
+
+    LaunchedEffect(pagerState.currentPage, pagerState.isScrollInProgress) {
+      if (!pagerState.isScrollInProgress)
+          currentSection = SavedHikesScreen.values()[pagerState.currentPage]
+    }
+
     Column(modifier = Modifier.fillMaxSize().padding(paddingValues)) {
-      Column(modifier = Modifier.weight(1f).testTag(TEST_TAG_SAVED_HIKES_SECTION_CONTAINER)) {
-        when (currentSection) {
-          SavedHikesScreen.Planned -> PlannedHikes(savedHikes)
-          SavedHikesScreen.Saved -> SavedHikes(savedHikes)
+      // Navigation items between nearby hikes, planned hikes, and saved hikes
+      SavedHikesTabsMenu(
+          selectedIndex = currentSection.ordinal, onSelectedChange = { currentSection = it })
+
+      HorizontalPager(
+          state = pagerState,
+          modifier = Modifier.fillMaxWidth().weight(1f),
+      ) { pageIndex ->
+        Column(modifier = Modifier.weight(1f).testTag(TEST_TAG_SAVED_HIKES_SECTION_CONTAINER)) {
```

Comment:
Question: Why is a weight necessary here?

Reply:
It is not. Thanks!